flask_project/config.py

```diff
@@ -0,0 +1,15 @@
+"""config.py - """
```

Comment:
Please add functionality to read configurations from environment variables with the same name during config initialization.